### PR TITLE
virsh_cpu_xml: add xml for s390x

### PR DESCRIPTION
**Pull request by smitterl**
Add incompatible cpu definition on s390x for expected error message.

Signed-off-by: Sebastian Mitterle <smitterl@redhat.com>

### CODE REVIEW

**Comment by smitterl** [comment]
JOB ID     : e68866a321d8a66ac8cac6bd54646e6dc6229419
JOB LOG    : /var/lib/avocado/job-results/job-2022-11-28T11.27-e68866a/job.log
 (1/1) type_specific.io-github-autotest-libvirt.virsh.cpu_xml.negative.hypervisor_cpu_compare.xml_declaration: PASS (16.30 s)
RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2022-11-28T11.27-e68866a/results.html
JOB TIME   : 16.76 s

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_cpu_xml.cfg [hunk 1]
```diff
@@ -56,7 +56,7 @@
                 - xml_declaration:
                     no cpu_baseline
                     file_xml_declaration = "<?xml version='1.0' encoding='UTF-8'?>"
-                    file_path = '../../../deps/capabilities.xml'
+                    file_path = '../../../deps/negative_domcapabilities_s390x.xml'
```
Comment (chunfuwen) [inline libvirt/tests/cfg/virsh_cmd/domain/virsh_cpu_xml.cfg:59]: do we need follow the convention to diff different arches ?:
  s390-virtio:
            file_path = '../../../deps/negative_domcapabilities_s390x.xml'

**Review by vasekhodina — APPROVED** [review]
LGTM

File: libvirt/tests/cfg/virsh_cmd/domain/virsh_cpu_xml.cfg (hunk 1 above)
Comment (smitterl) [reply libvirt/tests/cfg/virsh_cmd/domain/virsh_cpu_xml.cfg]: Yes, good catch, thank you. I updated this.

**Review by chunfuwen — APPROVED** [review]
LGTM

**Pull request merged**